.gitignore

```diff
@@ -1,178 +0,0 @@
-# Initially taken from GitHub's Python gitignore file
```

Comment:
Not sure why this was deleted but it caused the `.vscode` folder to get committed on your end

Reply:
Could you please revert this change?

Reply:
Sure.